packages/nvidia_nat_test/src/nat/test/tool_test_runner.py

```
@@ -153,6 +176,10 @@ def get_workflow_config(self) -> FunctionBaseConfig:
         """Mock implementation."""
         return FunctionBaseConfig()
 
+    def get_tools(self, tool_names: Sequence[str], wrapper_type):
```

Ruff (0.12.2)

Unused method argument: `tool_names`

(ARG002)